src/common/types.ts

```diff
@@ -377,12 +377,14 @@ export type GameAttributesLeague = {
 	difficulty: number;
 	divs: Div[];
 	draftType: DraftType;
+	draftAge: number[];
```

Comment:
Should be `draftAge: [number, number];` which tells TypeScript "this array always has exactly 2 numbers in it" rather than the current one that means "this is an array of numbers, of any length from 0 to infinity"

Reply:
Actually I'll fix this...